curriculum/js1/3.js

```diff
@@ -5,8 +5,11 @@
  * @returns {string}
  */
 
-const solution = (a, b) => {
-  return 0;
+const solution = (a,result="",i=0) => {
```

Comment:
whats the point of having an i counter? You can just decrement a to 0

Reply:
how do i decrement a. I don't know how to do that?

Reply:
i had i as a counter